### PR TITLE
systemd-boot-friend: update to 0.10.3

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

Update systemd-boot-friend.

Package(s) Affected
-------------------

systemd-boot-friend

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

### CODE REVIEW

File: extra-admin/systemd-boot-friend/autobuild/beyond [hunk 1]
```diff
@@ -1,2 +1,2 @@
-abinfo "Installing systemd-boot-friend-rs as systemd-boot-friend ..."
-mv -v "$PKGDIR/usr/bin/systemd-boot-friend-rs" "$PKGDIR/usr/bin/systemd-boot-friend"
+abinfo "Installing assets ..."
+PREFIX="$PKGDIR/usr/" ./install-assets.sh
```
Comment (WhiredPlanck) [inline extra-admin/systemd-boot-friend/autobuild/beyond:2]: May `"$SRCDIR"/install-assets.sh` here?
```suggestion
PREFIX="$PKGDIR/usr/" "$SRCDIR"/install-assets.sh
```

**Review by WhiredPlanck — CHANGES_REQUESTED** [review]

File: extra-admin/systemd-boot-friend/autobuild/beyond (hunk 1 above)
Comment (MingcongBai) [reply extra-admin/systemd-boot-friend/autobuild/beyond:2]: @WhiredPlanck is right.

Comment (OriginCode) [reply extra-admin/systemd-boot-friend/autobuild/beyond:2]: @MingcongBai BTW copied from ciel XD

**Comment by OriginCode** [comment]
New version released! XD

**Pull request closed without merging**